docs/experimental-features.md

```diff
@@ -439,20 +439,22 @@ When it is enabled:
 - IPNS publishers push records to a name-specific pubsub topic,
   in addition to publishing to the DHT.
 - IPNS resolvers subscribe to the name-specific topic on first
-  resolution and receive subsequently published records through pubsub in real time. This makes subsequent resolutions instant, as they are resolved through the local cache. Note that the initial resolution still goes through the DHT, as there is no message history in pubsub.
+  resolution and receive subsequently published records through pubsub in real time.
+  This makes subsequent resolutions instant, as they are resolved through the local cache.
 
 Both the publisher and the resolver nodes need to have the feature enabled for it to work effectively.
 
+Note: While IPNS pubsub has been available since 0.4.14, it received major changes in 0.5.0.
+Users interested in this feature should upgrade to at least 0.5.0
+
```

Comment:
Is it worth going into any more detail here about breaking changes between 0.4 and 0.5 (in particular the DHT switch from advertising CIDs to multihashes makes 0.4 and 0.5 IPNS over PubSub peers unlikely to find each other)?

Reply:
Eh, not really.